piccolo/columns/base.py

```diff
@@ -250,11 +250,11 @@ def get_default_alias(self):
 
         if self.call_chain:
             column_name = (
-                "$".join(
+                ".".join(
                     t.cast(str, i._meta.db_column_name)
                     for i in self.call_chain
                 )
-                + f"${column_name}"
+                + f".{column_name}"
             )
```

Comment:
There's a small risk changing this.

In the past, the following query:

```python
>>> await Band.select(Band.name, Band.manager.name)
[{'name': 'Pythonistas', 'manager.name': 'Guido'},
 {'name': 'Rustaceans', 'manager.name': 'Graydon'},
 {'name': 'C-Sharps', 'manager.name': 'Anders'}]
```

Executed this SQL:

```python
>>> print(Band.select(Band.name, Band.manager.name))
"""
SELECT ALL
    "band"."name" AS "name", 
    "band$manager"."name" AS "manager$name"    <- NOTE THIS
FROM "band"
    LEFT JOIN "manager" "band$manager"
    ON ("band"."manager" = "band$manager"."id")
"""
```

Note how we have`$` in the aliases - `AS "manager$name"`.

`$` was chosen because it's incredibly unlikely that someone would use `$` in their column names, and it still works even without the quotes around it:

```sql
SELECT name AS foo$bar from band;
-- Returns [{"foo$bar": "Pythonistas"}, ...]
```

We can use `.` instead, as long as we're careful to wrap all of our aliases in quotes (otherwise the query fails).

```sql
SELECT name AS foo.bar from band;
-- Error!
```

So our query now becomes:

```sql
SELECT ALL
    "band"."name" AS "name", 
    "band$manager"."name" AS "manager.name"    -- <- NOTE THIS
FROM "band"
    LEFT JOIN "manager" "band$manager"
    ON ("band"."manager" = "band$manager"."id")
```

This means we don't have to convert from `manager$name` to `manager.name` in Piccolo, which saves some overhead.

I checked the codebase, and it looks like we wrap all aliases in double quotes. If there's an edge case where we don't, it's something we'll have to fix.